code/game/objects/items/devices/instruments.dm

```diff
@@ -9,6 +9,7 @@
 	var/datum/song/handheld/song
 	var/instrumentId = "generic"
 	var/instrumentExt = "mid"
+	var/too_spooky = FALSE //this define if the instrument will spawn a copy upon dooting a new skeleton.

```

Comment:
This one is bit ugly. Do instruments even have any unique properties besides sound id and visuals so it can't be spectral/trumpet, spectral/saxophone etc ?

Reply:
They all behave the same, the only difference is which sound board they use.

Reply:
Could you make the spectralness a component please ? It just needs to do things on on hit after all, and wouldn't need any weird pathing or special variables  on base types that way. (And could be applied to anything for admin abuse)

Reply:
I'll do my best.